engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorImpl.java

```diff
@@ -956,8 +1006,18 @@ private int validatePropertyForNonDefaultGroup(ValueContext<?, Object> valueCont
 									.size() - numberOfConstraintViolationsBefore;
 						}
 					}
+
+					if ( typeUseConstraints.contains( metaConstraint ) ) {
+						boolean tmp = validateTypeConstraint( validationContext, valueContext, true, metaConstraint );
+						validationSuccessful = validationSuccessful && tmp;
+						if ( shouldFailFast( validationContext ) ) {
+							return validationContext.getFailingConstraints()
+									.size() - numberOfConstraintViolationsBefore;
+						}
+					}
 				}
-				if ( !validationSuccessful ) {
+
+				if (!validationSuccessful) {
```

Comment:
Interesting, is it even in accordance with the style guide? I assumed there'd always be empty spaces in braces?